__tests__/index.test.js

```diff
@@ -1,6 +1,7 @@
 const {NativeModules, NativeEventEmitter, Platform} = require("react-native");
 const {UnsupportedPlatformError} = require("../dist/errors");
 const {REFUND_REQUEST_STATUS} = require("../dist/purchases");
+const { PURCHASES_ARE_COMPLETED_BY, STOREKIT_VERSION } = require("@revenuecat/purchases-typescript-internal");
```

Comment:
So instead of importing these directly from the internal library, we can get them from `dist/purchases`, same as `REFUND_REQUEST_STATUS` I think. Mostly to keep consitency, also to make sure these classes are available directly from `react-native-purchases` and devs don't need to access the internal library.

Reply:
Yup yup, this! So when running this locally, you can run `npm run build` (or something like that) so that you have the `dist` directory with these resources in them 💪

Reply:
Thanks y'all! This makes total sense and those fields weren't being exposed yet, so great catch! This has been fixed now 😄